doc/classes/AnimationPlayer.xml

```diff
@@ -44,10 +44,10 @@
 				Returns the blend time (in seconds) between two animations, referenced by their keys.
 			</description>
 		</method>
-		<method name="get_method_call_mode" qualifiers="const" is_deprecated="true">
+		<method name="get_method_call_mode" qualifiers="const" deprecated="">
 			<return type="int" enum="AnimationPlayer.AnimationMethodCallMode" />
 			<description>
-				For backward compatibility. See [enum AnimationMixer.AnimationCallbackModeMethod].
+				See [enum AnimationMixer.AnimationCallbackModeMethod].
```

Comment:
Should this be the `deprecated=""` message instead, as "Use ... instead."?

Same for the others.